zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -559,21 +583,23 @@ def channel_message_to_zerver_message(realm_id: int, users: List[ZerverFieldsT],
                                       all_messages: List[ZerverFieldsT],
                                       zerver_realmemoji: List[ZerverFieldsT],
                                       zerver_subscription: List[ZerverFieldsT],
-                                      added_channels: AddedChannelsT,
+                                      added_channels: AddedChannelsT, id_list: List[int],
                                       domain_name: str) -> Tuple[List[ZerverFieldsT],
                                                                  List[ZerverFieldsT],
                                                                  List[ZerverFieldsT],
                                                                  List[ZerverFieldsT],
-                                                                 List[ZerverFieldsT]]:
+                                                                 List[ZerverFieldsT],
+                                                                 List[int]]:
     """
     Returns:
     1. zerver_message, which is a list of the messages
     2. zerver_usermessage, which is a list of the usermessages
     3. zerver_attachment, which is a list of the attachments
     4. uploads_list, which is a list of uploads to be mapped in uploads records.json
     5. reaction_list, which is a list of all user reactions
+    6. id_list, which is a list of all ids of messages, usermessages, reactions and attachments
```

Comment:
Is this all IDs, or the max IDs?

Reply:
Just max IDs, so that we know where to start the next IDs from.